aea/aea_builder.py

```diff
@@ -345,8 +349,9 @@ def _reset(self, is_full_reset: bool = False) -> None:
         self._build_called: bool = False
         if not is_full_reset:
             return
-        self._default_ledger = DEFAULT_LEDGER
-        self._default_connection: PublicId = DEFAULT_CONNECTION
+        self._default_ledger: Optional[str] = None
+        self._ledger_id_to_currency_denom: Dict[str, str] = {}
+        self._default_connection: Optional[PublicId] = None
```

Comment:
This is unified so that all defaults are None/empty

Reply:
ok